scipy/ndimage/_morphology.py

```diff
@@ -1217,7 +1217,7 @@ def grey_erosion(input, size=None, footprint=None, structure=None,
     if size is None and footprint is None and structure is None:
         raise ValueError("size, footprint, or structure must be specified")
 
-    return filters._min_or_max_filter(input, size, footprint, structure,
+    return _filters._min_or_max_filter(input, size, footprint, structure,
                                       output, mode, cval, origin, 1)
```

Comment:
Second line of arguments needs to be lined up.

Reply:
Nice catch! Fixed now.